exchanges/graphcache/src/store/data.ts

```diff
@@ -494,6 +489,10 @@ export const clearLayer = (data: InMemoryData, layerKey: number) => {
 
 /** Merges an optimistic layer of links and records into the base data */
 const squashLayer = (layerKey: number) => {
+  // Hide current dependencies from squashing operations
+  // const prevDependencies = currentDependencies;
+  // currentDependencies = new Set();
```

Comment:
Is this purposely left in?

Reply:
ah, I wanted to commit that, just a moment